cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -114,6 +114,11 @@ $ export TARGET_NAMESPACE=openshift-cnv CNV_CHANNEL=2.3 && oc patch -n "${TARGET
 +
 This command points your subscription to upgrade channel `2.3` and enables automatic updates.
 
+* In the virtual machine and virtual machine template wizards, *virtIO* is the default interface when attaching a CD-ROM. However, a VirtIO CD-ROM does not pass virtual machine validation and cannot be created.
```

Comment:
- It seems a little weird that it's `virtIO` (lowercase) in the web console, but `VirtIO` otherwise -- should that be changed?
- s/when attaching/when you attach (maybe)